### PR TITLE
Arm backend: Add tanh operator

### PR DESCRIPTION
**Pull request by SaoirseARM**
Adds Tanh operator to the Arm backend including unit tests

### CODE REVIEW

File: backends/arm/operators/op_tanh.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+def tanh_table_8bit(in_quantargs: QuantArgs, out_quantargs: QuantArgs):
+    """
+    Returns a table mapping 256 entries to tanh([qmin,qmax])
+    Reference: https://www.mlplatform.org/tosa/tosa_spec.html#_tanh
+    """
+
+    def tanh(x):
+        # Convert quantized input to floating point tanh input space.
+        v = dequantize_value(x, in_quantargs)
+        # Compute tanh.
```
Comment (digantdesai) [inline backends/arm/operators/op_tanh.py:76]: curious why not numpy.tanh or torch.nn.Tanh?

**Review by digantdesai — APPROVED** [review]

**Comment by digantdesai** [comment]
Seems like a flaky CI - rerunning it.

**Comment by digantdesai** [comment]
Rebase please? CI seems happy now

File: backends/arm/operators/op_tanh.py (hunk 1 above)
Comment (SaoirseARM) [reply backends/arm/operators/op_tanh.py:76]: Hi, 

Thanks for review! I just kept this consistent with sigmoid.: https://github.com/pytorch/executorch/pull/4114
I am happy to update this though if needed.

Thanks,
Saoirse

**Comment by SaoirseARM** [comment]
> Rebase please? CI seems happy now

Thanks, done!
